### PR TITLE
docs. alter-partition-verbose-result

### PR DESCRIPTION
**Pull request by den-crane**
Changelog category (leave one):
- Documentation (changelog entry is not required)

### CODE REVIEW

File: docs/ru/operations/settings/settings.md [hunk 1]
```diff
@@ -3808,3 +3808,38 @@ SELECT * FROM positional_arguments ORDER BY 2,3;
 
 Значение по умолчанию: `0`.
 
+## alter_partition_verbose_result {#alter-partition-verbose-result}
+
+Включает или отключает вывод информации о партах, к которым были успешно применены операции манипуляции с партициями и партами. Применимо к [ATTACH PARTITION|PART](../../sql-reference/statements/alteralter/partition.md#alter_attach-partition) и к [FREEZE PARTITION](../../sql-reference/statements/alteralter/partition.md#alter_freeze-partition)
```
Comment (alesapin) [inline docs/ru/operations/settings/settings.md:3813]:
```suggestion
Включает или отключает вывод информации о партах, к которым были успешно применены операции манипуляции с партициями и кусками. Применимо к [ATTACH PARTITION|PART](../../sql-reference/statements/alteralter/partition.md#alter_attach-partition) и к [FREEZE PARTITION](../../sql-reference/statements/alteralter/partition.md#alter_freeze-partition)
```

**Review by alesapin — APPROVED** [review]

**Comment by alesapin** [comment]
And what is wrong with `Docs check`?

**Comment by den-crane** [comment]
@Mergifyio rebase

**Comment by den-crane** [comment]
It was a typo "alteralter"
 "../../sql-reference/statements/alteralter/partition.md#alter_attach-partition"

**Pull request merged**